presto-main/src/main/java/io/prestosql/operator/scalar/JsonFunctions.java

```diff
@@ -360,17 +361,21 @@ public static Boolean jsonArrayContains(@SqlType(StandardTypes.JSON) Slice json,
         }
     }
 
+    @Deprecated
     @SqlNullable
     @ScalarFunction("json_array_get")
+    @Description("(DEPRECATED) If the extracted element is a string, it will be converted into an invalid JSON")
     @LiteralParameters("x")
     @SqlType(StandardTypes.JSON)
     public static Slice varcharJsonArrayGet(@SqlType("varchar(x)") Slice json, @SqlType(StandardTypes.BIGINT) long index)
     {
         return jsonArrayGet(json, index);
     }
 
+    @Deprecated
     @SqlNullable
     @ScalarFunction
+    @Description("(DEPRECATED) If the extracted element is a string, it will be converted into an invalid JSON")
     @SqlType(StandardTypes.JSON)
     public static Slice jsonArrayGet(@SqlType(StandardTypes.JSON) Slice json, @SqlType(StandardTypes.BIGINT) long index)
```

Comment:
This should be a separate change. Also, we should discuss whether we want to deprecate those functions or fix their behavior. So, let's start with an issue that describes how the behavior is broken and go from there.

Reply:
Ref: #2573